Dreary/README.md

```diff
@@ -0,0 +1,32 @@
+# Drery
+
+## Screenshots
+
+### BIB
+![BIB Screenshot](bib.png)
+
+### Psycho
+![Psycho Screenshot](psycho.png)
+
+### Deeper
+![Deeper Screenshot](deeper.png)
+
+### Mono
+![Mono Screenshot](mono.png)
+
+### Golden
+![Golden Screenshot](golden.png)
+
+##Important
+Sidebar Config option must be enabled for theme to work properly, you can use it without, however some features may not be available, such as edits to the sidebar.
```

Comment:
I may have missed this (since I think I already asked it to you) but what do you mean by this? Please clarify it in the README.